scripts/fobnail-mount.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description="Mount Fobnail-protected disk"
+After=network.target
+StopWhenUnneeded=yes
+
+[Service]
+EnvironmentFile=/etc/fobnail.cfg
+RuntimeDirectory=fobnail
+RuntimeDirectoryMode=0600
```

Comment:
Don't we need `0700` to be able to create file in that directory?

Reply:
As root we don't, but it should `0700` anyway, fixed,